### PR TITLE
HPCC-26996 Modern WU metrics tab consistency

### PR DESCRIPTION
**Pull request by GordonSmith**
The metrics tab was behaving differently to the rest of the WU Details tabs.

Signed-off-by: Gordon Smith <GordonJSmith@gmail.com>

## Type of change:
- [x] This change is a bug fix (non-breaking change which fixes an issue).
- [ ] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change improves the code (refactor or other change that does not change the functionality)
- [ ] This change fixes warnings (the fix does not alter the functionality or the generated code)
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [ ] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [ ] The commit message title makes sense in a changelog, by itself.
  - [x] The commit is signed.
- [ ] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [ ] I have read the CONTRIBUTORS document.
- [x] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [x] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Cloud-compatibility
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [x] The target branch of this pull request is appropriate for such a change.
- [x] There are no similar instances of the same problem that should be addressed
  - [x] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [x] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browsers
  - [ ] The component(s) render as expected

## Smoketest:
- [ ] Send notifications about my Pull Request position in Smoketest queue.
- [ ] Test my draft Pull Request.

### CODE REVIEW

File: esp/src/src-react/routes.tsx [hunk 1]
```diff
@@ -83,10 +83,8 @@ export const routes: RoutesEx = [
...
             { path: "/:Wuid", action: (ctx, params) => import("./components/WorkunitDetails").then(_ => <_.WorkunitDetails wuid={params.Wuid as string} />) },
-            { path: "/:Wuid/outputs/:Name", action: (ctx, params) => import("./components/Result").then(_ => <_.Result wuid={params.Wuid as string} resultName={params.Name as string} filter={parseSearch(ctx.search) as any} />) },
-            { path: "/:Wuid/metrics", action: (ctx, params) => import("./components/Metrics").then(_ => <_.Metrics wuid={params.Wuid as string} />) },
-            { path: "/:Wuid/metrics/:id", action: (ctx, params) => import("./components/Metrics").then(_ => <_.Metrics wuid={params.Wuid as string} selection={(params.id as string)} />) },
             { path: "/:Wuid/:Tab", action: (ctx, params) => import("./components/WorkunitDetails").then(_ => <_.WorkunitDetails wuid={params.Wuid as string} tab={params.Tab as string} />) },
+            { path: "/:Wuid/:Tab/:State", action: (ctx, params) => import("./components/WorkunitDetails").then(_ => <_.WorkunitDetails wuid={params.Wuid as string} tab={params.Tab as string} state={(params.State as string)} />) },
```
Comment (jeclrsg) [inline esp/src/src-react/routes.tsx:87]: This one should precede the more generic /:Wuid/:Tab route

**Review by jeclrsg — CHANGES_REQUESTED** [review]
@GordonSmith Looks good. Only one issue with the route ordering

File: esp/src/src-react/routes.tsx (hunk 1 above)
Comment (GordonSmith) [reply esp/src/src-react/routes.tsx:87]: FWIW the additional "/" is enough to differentiate them, but for consistency I agree.

Comment (GordonSmith) [reply esp/src/src-react/routes.tsx:87]: @jeclrsg just checked the rest of the file and this code is consistent with the rest, so I am inclined to leave as is for now.  Can you re-review?

**Review by jeclrsg — APPROVED** [review]

**Pull request merged**